<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEGUIWidget</class>
 <widget class="QWidget" name="CEGUIWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CEGUIGraphicsView" name="view">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="viewportUpdateMode">
      <enum>QGraphicsView::MinimalViewportUpdate</enum>
     </property>
     <property name="widgetResizable" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomLine" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>6</pointsize>
          <italic>true</italic>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Embedded CEGUI</string>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCursorPosition">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblZoom">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="resolutionBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <item>
         <property name="text">
          <string>Project default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800x600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1024x768</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280x720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280x1024</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1650x1080</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920x1080</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="debugInfoButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../data/Resources.qrc">
          <normaloff>:/icons/cegui/debug_info.png</normaloff>:/icons/cegui/debug_info.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHelp">
        <property name="toolTip">
         <string>Show help overlay</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../data/Resources.qrc">
          <normaloff>:/icons/actions/help_about.png</normaloff>:/icons/actions/help_about.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CEGUIGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>src/ui/CEGUIGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../data/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
